packages/govuk-frontend/src/govuk/components/accordion/accordion.mjs

```diff
@@ -395,7 +411,22 @@ export class Accordion extends GOVUKFrontendComponent {
     const $button = $section.querySelector(`.${this.sectionButtonClass}`)
     const $content = $section.querySelector(`.${this.sectionContentClass}`)
 
-    if (!$showHideIcon || !$showHideText || !$button || !$content) {
+    if (!$button) {
+      throw new ElementError({
+        componentName: 'Accordion',
+        identifier: `Section button (\`<span class="${this.sectionButtonClass}">\`)`
```

Comment:
**nitpick** Technically at that stage, it's an actual `button` tag, so we could either:

```suggestion
        identifier: `Section button (\`<button class="${this.sectionButtonClass}">\`)`
```

Which would help people look for a button, though what they likely need to do for fixing is making sure the placeholder is in so we turn it into a button with:

```suggestion
        identifier: `Section button placeholder (\`<span class="${this.sectionButtonClass}">\`)`
```

Reply:
Thanks for spotting this

It's a quirk of Accordion that's caught me out too, that `this.sectionButtonClass` matches different things

Should we instead return early for `!$button` here, knowing we've thrown already for the placeholder?

Reply:
Good idea! At this stage, we're the ones that created the `<button>` so we can early return like in other listener for now 😊

Reply:
Thanks for taking a look, folks. I've now made these tweaks in the latest commit.